garage/np/algos/batch_polopt.py

```diff
@@ -1,135 +1,84 @@
-from dowel import logger, tabular
+from collections import deque
 
-from garage.experiment import snapshotter
+from dowel import tabular
+import numpy as np
+
+from garage.misc import special
 from garage.np.algos.base import RLAlgorithm
-from garage.plotter import Plotter
-from garage.sampler import BatchSampler
 
 
 class BatchPolopt(RLAlgorithm):
     """
-    Base class for batch sampling-based policy optimization methods.
-    This includes various policy gradient methods like vpg, npg, ppo, trpo,
-    etc.
+    Base class for cem and cam_es.
+
+    Args:
+        env_spec (garage.envs.EnvSpec): Environment specification.
+        policy (garage.tf.policies.base.Policy): Policy.
```

Comment:
does it actually use policies and baselines from garage.tf?

Reply:
Yes.